options/train_options.py

```diff
@@ -46,5 +46,8 @@ def initialize(self, parser):
         parser.add_argument('--use_contrastive_loss_D', action='store_true')
         parser.add_argument('--ddp_port', type=str, default='12355')
 
+        #imgaug options
+        parser.add_argument('--diff_aug_policy',type=str, default='',help='choose between color translation and cutout')
```

Comment:
multiple choice allowed & comma separated?

Reply:
no you have to choose only one policy !

Reply:
It looks like your code allow multiple comma-separated policies, see util/diff_aug.py:9